pkg/middlewares/ratelimiter/redisrate/rate.go

```diff
@@ -0,0 +1,151 @@
+package redisrate
+
+import (
+	"context"
+	"fmt"
+	"math"
+	"strconv"
+	"time"
+
+	"github.com/redis/go-redis/v9"
+	"golang.org/x/time/rate"
+)
+
+const redisPrefix = "rate:"
+
+// Inf is the infinite rate limit; it allows all events (even if burst is zero).
+const Inf = rate.Limit(math.MaxFloat64)
+
+// InfDuration is the duration returned by Delay when a Reservation is not OK.
+const InfDuration = time.Duration(math.MaxInt64)
```

Comment:
I think these 2 constants are not used anymore.

Reply:
Agree